### PR TITLE
added statement of need to paper

### PR DESCRIPTION
**Pull request by zonca**
added statement of need to paper: it is necessary for the JOSS paper

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -34,11 +34,17 @@ date: 22 July 2021
 bibliography: paper.bib
 ---
 
+# Statement of Need
+
+The most fruitful region of the radio spectrum to study the Early Universe is the microwave region. At those frequencies peaks the Cosmic Microwave Background (CMB) eletromagnetic radiation coming directly from the beginning of the Universe, just 370 thousand years into the 13.8 billion years age of the Universe.
```
Comment (bthorne93) [inline paper/paper.md:39]:
```suggestion
The Cosmic Microwave Background (CMB) radiation, emitted just 370 thousand years after the Big Bang, is a pristine probe of the Early Universe. After being emitted at hight temperatures, the CMB was redshifted by the subsequent 13.8 billion years of cosmic expansion, such that it is brightest at microwave frequencies today.
```

File: paper/paper.md [hunk 2]
```diff
@@ -34,11 +34,17 @@ date: 22 July 2021
...
+However, our own Milky Way galaxy also emits in the microwaves, for example through thermal radiation by interstellar dust grains or through synchrotron emission by relativistic electrons spiralling in magnetic fields.
```
Comment (bthorne93) [inline paper/paper.md:40]:
```suggestion
However, our own Milky Way galaxy also emits in the microwave, obscuring our view of the CMB.  Examples of this emission are thermal radiation by interstellar dust grains, and synchrotron emission by relativistic electrons spiralling in magnetic fields.
```

File: paper/paper.md [hunk 3]
```diff
@@ -34,11 +34,17 @@ date: 22 July 2021
...
+Cosmologists need to create syntetic maps of the CMB and of the galactic emission based on available data and on models that properly scale them to different frequencies. Those maps are useful to test data reduction algorithms, to understand residual systematics, to forecast maps produced by future instruments, to run Montecarlo analysis for noise estimation and more.
```
Comment (bthorne93) [inline paper/paper.md:41]:
```suggestion
Cosmologists need to create synthetic maps of the CMB and of the galactic emission based on available data and on physical models that extrapolate observations to different frequencies. The resulting maps are useful to test data reduction algorithms, to understand residual systematics, to forecast maps produced by future instruments, to run Monte Carlo analysis for noise estimation, and more.
```

File: paper/paper.md (hunk 2 above)
Comment (zonca) [reply paper/paper.md:40]: should it be "in the microwaves" or in the microwave region?

Comment (bthorne93) [reply paper/paper.md:40]: I would usually say "in the microwave", but perhaps that is a bit colloquial. "In the microwave portion of the spectrum" would be less colloquial. I would avoid "in the microwave region" as it's not clear we're referring to a region of a spectrum, and I don't think "in the microwaves" is correct.

**Comment by bthorne93** [comment]
Noticed a typo in my own edit: "Emitted at hight temperatures" should be corrected. Sorry!

File: paper/paper.md (hunk 2 above)
Comment (zonca) [reply paper/paper.md:40]: "In the microwave portion of the spectrum"

**Pull request merged**